pkg/providers/instance/instance.go

```diff
@@ -86,7 +86,7 @@ type Provider struct {
 	launchTemplateProvider *launchtemplate.Provider
 	loadBalancerProvider   *loadbalancer.Provider
 	resourceGroup          string
-	subnetID               string
+	defaultSubnetID        string
```

Comment:
should we remove this and just use the env variable instead? How do we feel about these global access patterns with the environment variable?